Automattic/jetpack
# Jetpack: Check that the Sync_Actions::do_only_first_initial_sync method exists

#### Changes proposed in this Pull Request:
* Check that the `Sync_Actions::do_only_first_initial_sync` method exists before calling it. This will prevent fatal errors that can occur when these conditions are both true:
   * Another connected and synced plugin is installed on the site, and that plugin uses a
   version of the Sync package that doesn't have the `do_only_first_initial_sync` method.
   * Jetpack is activated via a method that the autoloader can't detect, such as WP CLI.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. Install this branch of Jetpack. Don't activate it yet.
2. Install version 1.0.0 of Jetpack Backup. This version does not have the `Sync_Actions::do_only_first_initial_sync` method.
3. Connect Jetpack backup.
4. Activate Jetpack via WP CLI: `wp plugin activate jetpack`.
5. Verify that Jetpack was successfully activated and that no fatal errors occurred.
